model.py

```diff
@@ -168,6 +168,11 @@ def show(self, p, x, y, v=None, res=True, x2=None, dx=None):
             #gplot.y2range('[-0.2:2]').ytics('nomirr').y2tics()
             #args += (",", x, y-ymod, x2, "us lam?3:1:2 w p pt 7 ps 0.5 lc 1 axis x1y2 t 'res (%.3g \~ %.3g%%)', 0 lc 3 axis x1y2 t ''" % (rms, rms/np.mean(ymod)*100))
             args += (",", x, y-ymod, x2, "us lam?3:1:2 w p pt 7 ps 0.5 lc 1 t 'res (%.3g \~ %.3g%%)', 0 lc 3 t ''" % (rms, rms/np.mean(ymod)*100))
+        if notell:	# plot telluic free spectrum
+            rms = np.std((y/ymod-1)*np.nanmean(y))
+            prms = rms / np.mean(ymod) * 100
+            gplot.mxtics().mytics().my2tics()
+            args += (",", x, (y/ymod-1)*np.nanmean(y), x2, "us lam?3:1:2 w l lc 1 t 'res (%.3g \~ %.3g%%)', 0 lc 3 t ''" % (rms, rms/np.mean(ymod)*100))
```

Comment:
The last argument is just prms, which was just assigned before.

Reply:
It basically plots the telluric-corrected spectrum. Or spectrum with no tellurics. (I had no better name for it, but I am open for suggestions)
The output from this option would produce a plot like the following. Here the red line (the "residuals") is the telluric corrected spectrum. The blue points are the modeled telluic spectrum.

[viper_GJ588_o12_tell](https://user-images.githubusercontent.com/77435060/200579440-a2c24f40-b07b-41b7-af7c-b74b8fdc2481.png)
)

Reply:
Ok, I see. The plot is very helpful.
Seems the function is a bit misused/enhanced. I'm fine with it. Another names? Not sure, `tell_cleaned`?
How about `np.mean(ymod)` instead of `np.nanmean(y)`.

Only loud thinking. Instead an option `rel_res` (relative residuals). It could be also a float value used to for scaling.

Reply:
I guess `tell_cleaned` would be an option. But in theory it doesn't has to be just tellurics. It also f.e. could be tellurics+cell removed. Is this what you mean by `rel_res`? But I don't understand where you would like to put a scaling in. 
In theory I am fine with any name, as long as it is the same everywhere and the code works ;-) I am never good with choosing names.

> How about np.mean(ymod) instead of np.nanmean(y).
You mean for the plotting of the non-corrected spectrum?

Reply:
then maybe `absorp_clean` or `rel_fac`:

```python
def show(self, p, ..., rel_fac=None):
        '''
        res: Show residuals.
        x2: Values for second x axis.
        rel_fac: Factor for relative residuals.
        '''
        ...
        if res or rel_fac:
            ...
            # linear or relative residuals
            col2 = rel_fac * (y/ymod - 1) if rel_fac else y - ymod
            args += (",", x, col2, x2, ...)
        ...
```

Reply:
Well, I like the idea of putting both options together and I was just trying to implement it until I faced  two problems:
1) in this way I am not able to set `rel_fac` to `np.mean(ymod)` as I would have to give that input in viper.py. But the modeling and plotting happens in model.py. I therefore have decided to use `col2 = rel_fac * np.mean(ymod) * (y/ymod - 1) if rel_fac else y - ymod`
2) Beside of the slightly different calculation, also the plotting varies for both options. For `res` we have points, while for `rel_fac` we plot lines. So I will include also a separation for the plotting itself.